tests/drivers/dac/dac_api/src/test_dac.c

```diff
@@ -87,6 +87,18 @@
 #define DAC_RESOLUTION		12
 #define DAC_CHANNEL_ID		0
 
+#elif defined(CONFIG_BOARD_RD_RW612_BGA)
+
+#define DAC_DEVICE_NODE		DT_NODELABEL(dac0)
+#define DAC_RESOLUTION		10
+#define DAC_CHANNEL_ID		0
+
+#elif defined(CONFIG_BOARD_FRDM_MCXN947)
```

Comment:
Please add this to the already existing group with same configuration above.

Reply:
updated.